.github/workflows/iroha2-dev-pr.yml

```diff
@@ -61,9 +62,12 @@ jobs:
         run: |
           rustup component add llvm-tools-preview
           rustup +$NIGHTLY component add llvm-tools-preview
+          mold --run cargo --version
       - uses: Swatinem/rust-cache@v2
       - name: Run tests, with ${{ matrix.feature_flag }}
         run: |
+          mold --run cargo --version
+          mold --run cargo test -p iroha_primitives --test ui
```

Comment:
The only reason to keep this PR as a draft is this:
https://github.com/hyperledger/iroha/runs/8053570359?check_suite_focus=true#step:6:14
1.63 behaves as if it is not 1.63 and fails the UI test. Locally it succeeds. Do you have any idea what's going on @appetrosyan  @BAStos525

Reply:
I'm honestly not a huge fan of how we use `trybuild`. We should not rely on the error messages to be precisely a specific error message. Besides I think we're using primitives long-enough hence the trybuild tests in the `iroha_primitives` crate can be safely ignored.